packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/request.js

```diff
@@ -112,19 +112,23 @@ describe('plugin-meetings', () => {
         const deviceUrl = 'deviceUrl';
         const correlationId = 'random-uuid';
         const roapMessage = 'roap-message';
+        const permissionToken = 'permission-token';
 
         await meetingsRequest.joinMeeting({
           locusUrl,
           deviceUrl,
           correlationId,
-          roapMessage
+          roapMessage,
+          permissionToken
```

Comment:
would be good to have a test checking that inviteeAddress is handled correctly and also that we fallback to using meetingNumber